core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -645,9 +646,24 @@ public ServerBuilder gracefulShutdownTimeout(Duration quietPeriod, Duration time
     /**
      * Sets the {@link Executor} dedicated to the execution of blocking tasks or invocations.
      * If not set, {@linkplain CommonPools#blockingTaskExecutor() the common pool} is used.
+     *
+     * @deprecated Use {@link #blockingTaskExecutor(Executor, boolean)}.
+     *
      */
+    @Deprecated
```

Comment:
I'm wondering if it's a good idea to deprecate this. There should be some users who do not want to care about `shutdownOnStop`. How about turning this back from deprecated? /cc @trustin

Reply:
Just like other similar builder methods:

- `ServerBuilder.workerGroup()`
- `ServerBuilder.accessLogger()`

I think we do not really know which behavior a user prefers when it comes down to life cycle. People who do not care could make an informed decision of specifying `false` in my opinion.